ocrd/ocrd/mets_server.py

```diff
@@ -0,0 +1,253 @@
+"""
+# METS server functionality
+"""
+import re
+from os import environ, _exit
+from io import BytesIO
+from typing import Any, Dict, Optional, Union, List, Tuple
+
+from fastapi import FastAPI, Request, File, Form, UploadFile
+from fastapi.responses import JSONResponse
+from requests import request, Session as requests_session
+from requests_unixsocket import Session as requests_unixsocket_session
+from pydantic import BaseModel, Field, constr, ValidationError
+
+import uvicorn
+
+from ocrd_models import OcrdMets, OcrdFile
+from ocrd_utils import initLogging, getLogger, deprecated_alias
+
+#
+# XXX HACKS TODO
+#
+initLogging()
+
+#
+# Models
+#
+
+class OcrdFileModel(BaseModel):
+    file_grp : str = Field()
+    file_id : str = Field()
+    mimetype : str = Field()
+    page_id : Union[str, None] = Field()
+    url : str = Field()
+
+    @staticmethod
+    def create(file_grp : str, file_id : str, page_id : Union[str, None], url : str):
```

Comment:
```suggestion
    def create(file_grp : str, file_id : str, page_id : Union[str, None], url : str, mimetype=mimetype):
```
I think you missed that when refactoring.

Reply:
I did, thanks for spotting. 3532600b1